src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -85,7 +85,13 @@
                 "weightCouldNotBeUpdated": "Die Gewichtung für den Testfall {{id}} konnte nicht übernommen werden: {{error}}.",
                 "saveTestCases": "Testfälle speichern",
                 "testCasesUpdated": "Die Testfälle wurden erfolgreich gespeichert.",
-                "testCasesCouldNotBeUpdated": "Die folgenden Testfälle konnten nicht gespeichert werden: {{testCases}}."
+                "testCasesCouldNotBeUpdated": "Die folgenden Testfälle konnten nicht gespeichert werden: {{testCases}}.",
+                "noUnsavedChanges": "Keine ungespeicherten Änderungen",
+                "unsavedChanges": "Ungespeicherte Änderungen",
+                "updatedTestCasesShort": "Veränderte Testfälle",
+                "updatedTestCases": "Achtung: Du hast die Testfälle der Aufgabe verändert, aber nicht die Einrichungen der Studenten mit den aktuellen Einstellungen erneut ausgeführt. Wollen Sie wirklich den Bereich verlassen?",
+                "updatedTestCasesTooltip": "Du hast die Testfälle verändert, allerdings hast diese Aktion keine Auswirkungen auf die bereits vorhandenen Ergebnisse der Studenten. Über den Button rechts neben diesem Symbol können Sie die Einreichungen aller Studenten mit den veränderten Testfällen erneut bewerten.",
```

Comment:
```suggestion
                "updatedTestCasesTooltip": "Du hast die Testfälle verändert, allerdings hat diese Aktion keine Auswirkungen auf die bereits vorhandenen Ergebnisse der Studenten. Über den Button 'Alle erneut ausführen' kannst du die Einreichungen aller Studenten mit den veränderten Testfällen erneut bewerten.",
```

Reply:
You are right, I'll change it as suggested.